src/Resources/skeleton/message/Message.tpl.php

```diff
@@ -0,0 +1,32 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace; ?>;
+
+class <?= $class_name."\n" ?>
+{
+    /*
+     * Add whatever properties & methods you need to hold the
+     * data for this message class.
+     *
+     * Note - if you're handling this message async, you may need
+     * a getter & setter method for each property so that it serializes
+     * and unserializes correctly.
+     */
+
+//     private $name;
+//
+//     public function __construct(string $name)
+//     {
+//         $this->name = $name;
+//     }
+//
+//    public function getName(): string
+//    {
+//        return $this->name;
+//    }
+//
+//    public function setName(string $name)
```

Comment:
Would be preferable to keep messages immutable. What about not promoting setters?

Reply:
Good idea 💡

Reply:
I agree to remove the setter example